content/en/real_user_monitoring/guide/proxy-rum-data.md

```diff
@@ -12,10 +12,42 @@ further_reading:
 ## Overview
 
 The RUM Browser SDK can be configured to send requests through a proxy. Requests made still need to be forwarded to Datadog.
+When you set the `proxy` [initialization parameter][1], all RUM data is sent to the specified URL using the POST method (for example, `https://www.proxy.com/foo`).
 
-## SDK initialization
+## Prerequisites
 
-When you set the `proxy` [initialization parameter][1], all RUM data is sent to the specified URL using the POST method (for example, `https://www.proxy.com/foo`).
+A request to datadog (ex: `https://browser-intake-datadoghq.eu/api/v2/rum?ddsource=browser&...`) is composed of three parts:
+- The Datadog intake origin corresponding to your `site` [initialization parameter][1] (ex: `https://browser-intake-datadoghq.eu`)
+- The path containing the API version and the product (ex: `/api/v2/rum`)
+- The parameters (ex: `ddsource=browser&...`)
+
+The Datadog intake origin corresponding to your site parameter should be defined in your proxy implementation. The path and parameters should be attached to the requests to the proxy.
+Datadog intake origins for each site are listed below:
+
+| Site    | Site Parameter            | Datadog intake origin                      |
+|---------|---------------------------|--------------------------------------------|
+| US1     | `datadoghq.com` (default) | `https://browser-intake-datadoghq.com`     |
+| US3     | `us3.datadoghq.com`       | `https://browser-intake-us3-datadoghq.com` |
+| US5     | `us5.datadoghq.com`       | `https://browser-intake-us5-datadoghq.com` |
+| EU1     | `datadoghq.eu`            | `https://browser-intake-datadoghq.eu`      |
+| US1-FED | `ddog-gov.com`            | `https://browser-intake-ddog-gov.com`      |
+| AP1     | `ap1.datadoghq.com`       | `https://browser-intake-ap1-datadoghq.com` |
+
+To successfully proxy request to Datadog:
+
+1. Build the final Datadog intake URL by using:
+   - the Datadog intake origin corresponding to your site.
+   - the path and parameters from the request to the proxy.
+2. Add a `X-Forwarded-For` header containing the request client IP address for accurate geoIP.
+3. Forward the request to the Datadog intake URL using the POST method.
+4. Leave the request body unchanged.
+
+The request body can contain binary data and should not be converted to a string. Make sure your proxy implementation forwards the raw body without conversion.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**  It could be nice to highlight this sentence as it's something error prone in proxy config (and even more now since we will compress RUM data)